.github/scripts/hermetic_library_generation.sh

```diff
@@ -84,16 +84,29 @@ git show "${target_branch}":"${generation_config}" > "${baseline_generation_conf
 # get .m2 folder so it's mapped into the docker container
 m2_folder=$(dirname "$(mvn help:evaluate -Dexpression=settings.localRepository -q -DforceStdout)")
 
+# download api definition from googleapis repository
+googleapis_commitish=$(grep googleapis_commitish "${generation_config}" | cut -d ":" -f 2 | xargs)
+api_def_dir=$(mktemp -d)
+git clone https://github.com/googleapis/googleapis.git "${api_def_dir}"
+pushd "${api_def_dir}"
+git checkout "${googleapis_commitish}"
+popd
+
 # run hermetic code generation docker image.
 docker run \
   --rm \
   -u "$(id -u):$(id -g)" \
   -v "$(pwd):${workspace_name}" \
   -v "${m2_folder}":/home/.m2 \
+  -v "${api_def_dir}:${workspace_name}/api" \
```

Comment:
This is essentially googleapis folder, maybe we can just use `/googleapis` to make it easier to understand?

Reply:
I try to use a generalized name here to specify that the script can generate from any protos, not necessarily from googleapis.

Reply:
The docker image can be used by any protos, but I think `hermetic_library_generation.sh` is only going to be used for generating googleapis. For generating other protos, the volumn name actually doesn't really matter, it's good as long as the volumn and api-definition-path match.
That being said, it's fine if you want to make it generic, but maybe change it to `apis` to imply that it could contain multiple.

Reply:
> but I think hermetic_library_generation.sh is only going to be used for generating googleapis

Makes sense, I changed it to `googleapis`.